template/.github/workflows/deploy.yml

```
@@ -0,0 +1,33 @@
+name: Netlify Release
+on:
+  push:
+    branches: ['main', 'master']
+  workflow_dispatch:
+    inputs:
+      deploy-msg:
+        description: 'Deploy Message (visible in Netlify)'
+        required: true
+
+jobs:
+  deploy:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install modules
+        run: yarn
+
+      - name: Build
+        run: yarn build
+
+      - name: Deploy to Netlify
+        uses: nwtgck/actions-netlify@v1.2
+        with:
+          publish-dir: './build'
+          deploy-message: "Deploy from GitHub Actions. ${{ github.event.inputs.deploy-msg }}"
```

Should we change to using colon here?
```suggestion
          deploy-message: "Deploy from GitHub Actions: ${{ github.event.inputs.deploy-msg }}"
```